<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Zastavka</class>
 <widget class="QWidget" name="Zastavka">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QProgressBar{
    border: 2px solid grey;
    border-radius: 5px;
    text-align: center
}

QProgressBar::chunk {
    background-color: black;
    width: 10px;
    margin: 1px;
}</string>
  </property>
  <widget class="QLabel" name="fon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resource/images.qrc">:/newPrefix/mishka.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>601</width>
     <height>51</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="WaitLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="zagruziButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>250</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(255, 255, 255);
width:60px;
height:100px;
font-size:14px;
border:none;
text-align:center
}

QPushButton:hover{
	background-color: rgb(239, 239, 239);
}

QPushButton:pressed{
	
	background-color: rgb(255, 224, 245);
}</string>
   </property>
   <property name="text">
    <string>Загрузить</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resource/images.qrc"/>
 </resources>
 <connections/>
</ui>
